### ISSUE TITLE
Merge outlines of merged files

### ISSUE THREAD

**Issue by junrrein** [junrrein/pdfslicer#156]
Currently, Slicer carries over the outline of the first opened file.

It would be nice if we could merge the outlines from added files, too.

Currently, QPDF doesn't offer that capability, but this would be nice to have in the future.

**Comment by rugk** [comment]
Indeed, currently it just keeps the one from the first document though.
